deploy/902_set_asset_admin.ts

```diff
@@ -0,0 +1,32 @@
+import {DeployFunction} from 'hardhat-deploy/types';
+
+const func: DeployFunction = async function (hre) {
+  const {deployments, getNamedAccounts} = hre;
+  const {log, execute, read} = deployments;
+  const {assetAdmin, assetBouncerAdmin} = await getNamedAccounts();
+
+  const currentAdmin = await read('Asset', 'getAdmin');
+  if (currentAdmin.toLowerCase() !== assetAdmin.toLowerCase()) {
+    await execute(
+      'Asset',
+      {from: currentAdmin, log: true},
+      'changeAdmin',
+      assetAdmin
+    );
+  }
+
+  const currentBouncerAdmin = await read('Asset', 'getBouncerAdmin');
+  if (currentBouncerAdmin.toLowerCase() !== assetBouncerAdmin.toLowerCase()) {
+    log('setting Sand Execution Admin');
+    await execute(
+      'Asset',
+      {from: currentBouncerAdmin, log: true},
+      'changeBouncerAdmin',
+      assetBouncerAdmin
+    );
+  }
+};
+
+export default func;
+func.runAtTheEnd = true;
+func.dependencies = ['Asset'];
```

Comment:
for setting admin the tag need to be `['Asset', 'Asset_setup']` and dependencies : `Asset_deploy`